frontend/src/pages/GraphPF/GraphPFElems.tsx

```diff
@@ -508,14 +514,15 @@ const toFixedErrRate = (num: number): string => {
   return `${trimFixed(num.toFixed(num < 1 ? 1 : 0))}%err`;
 };
 
-const toFixedByteRate = (num: number, includeUnits: boolean): string => {
+const toFixedByteRate = (num: number, includeUnits: boolean, includeMarker?: string): string => {
```

Comment:
This `includeMarker` param is unused given the commented code.  But it's fine if you think you may use it later.

Reply:
This was used to show this * when the rates was the sum of the source and destination rates, as the rates is a marker, I added this additional parameter, but is not used anymore, removed.